### ISSUE TITLE
add search for counter

### ISSUE THREAD

**Issue by SrPatinhas** [DJScias/LoLES#12]
https://chrome.google.com/webstore/detail/league-of-legends-searche/ndadondgnhbodkhnhcnagmjnanecnogl

**Comment by DJScias** [comment]
Don't we somewhat support this due to the fact we offer support for http://www.lolcounter.com/ ?

**Comment by SrPatinhas** [comment]
yes, but we we can just add another input to be a more quick way (need to check later)

**Comment by DJScias** [comment]
Sure, let me know if you figure something out.

**Comment by DJScias** [comment]
What about http://www.counterstats.net/ ? Seems relatively new and it's part of the Mobafire network.

I'm working on something regarding this site.
